lib/streamlit/elements/widgets/selectbox.py

```diff
@@ -312,31 +446,41 @@ def _selectbox(
 
         if index is not None and len(opt) > 0 and not 0 <= index < len(opt):
             raise StreamlitAPIException(
-                "Selectbox index must be greater than or equal to 0 and less than the length of options."
+                "Selectbox index must be greater than or equal to 0 "
+                "and less than the length of options."
             )
 
         session_state = get_session_state().filtered_state
         if key is not None and key in session_state and session_state[key] is None:
             index = None
 
+        (formatted_options, formatted_option_to_option_index) = create_mappings(
```

Comment:
nit: why does it need the brackets here? wouldn't `formatted_options, formatted_option_to_option_index = create_mappings(` also work?

Reply:
yeah you are right, I will remove them 👍